### DISCUSSION TITLE
ARK ASA Token InGame Command

### DISCUSSION THREAD

**Discussion by PlayButtonYT** [gameserverapp/Platform#783] [Suggestions]
Hello everyone, 
I wanted to politely ask if it's possible to add an in-game command to the GSA Mod to send tokens directly to the website account, for example, add.Token XXX. 
I understand that console players are not allowed to possess in-game currencies through mods anymore, but this wouldn't be direct ownership. Furthermore, the achievement system and playtime with tokens for console players still work. 
Perhaps it would be smarter to run such a command through a GSA plugin or through your internal server software management to avoid interference from Curseforge.

Currently, I'm working on a quest system for my server that operates through a server plugin (Lethal Quests). Here, I wanted to reward players with tokens for completed quests, including console players, of course. 
There are certainly ways to adjust the system so that Curseforge and console manufacturers cannot influence it. 
Maybe you could think about this. 

Currently, this is really dividing the community, and the revenue for private servers like mine has dropped significantly since the ban. Additionally, I have the problem, as do many others, that if I have an in-game shop installed, console players can access it, but players cannot place mod objects because crossplay-compatible mods are not installed. This frustrates many players greatly, and I can understand this! But as I said, please consider this, and I would truly appreciate any proposed solution to be able to serve console players and PC players equally on my server again. 

Thank you very much, your big fan JaSha from PlayButton.

**Comment by mvaessen** [comment]
Hi, this section is for request games :). As mentioned on discord earlier; tokens do not exist on the mod side of things. If you like to see something like this available, I recommend posting a suggestion (in the right section) involving the exposure of api endpoints that the gsa mod is using for token gems.
